src/doc_scan/request/check/sandbox.third.party.identity.check.builder.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const SandboxDocumentCheckBuilder = require('./sandbox.document.check.builder');
+const SandboxCheckReport = require('./sandbox.check.report');
+const SandboxCheckResult = require('./sandbox.check.result');
+const SandboxThirdPartyIdentityCheck = require('./sandbox.third.party.identity.check');
+
+class SandboxThirdPartyIdentityCheckBuilder extends SandboxDocumentCheckBuilder {
+  /**
+   * @returns {SandboxIdDocumentComparisonCheck}
```

Comment:
To update

Reply:
Thanks for pointing this out. Will update 🙂